public/app/features/playlist/PlaylistNewPage.test.tsx

```diff
@@ -37,15 +39,11 @@ function getTestContext({ name, interval, items }: Partial<Playlist> = {}) {
 }
 
 describe('PlaylistNewPage', () => {
-  beforeEach(() => {
-    jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
(This is just something else I spotted, we don't need to spy on the console, we can get rid of the error from the test by awaiting the element instead)